doc/api-extensions.md

```diff
@@ -2289,3 +2289,7 @@ Check the list of {doc}`configuration options </config-options>` to see which co
 ## `syslog_socket`
 
 This introduces a syslog socket that can receive syslog formatted log messages. These can be viewed in the events API and `lxc monitor`, and can be forwarded to Loki. To enable this feature, set `core.syslog_socket` to `true`.
+
+## `instances_nic_limits_priority`
+
+This introduces a new per-NIC `limits.priority` option.
```

Comment:
We could add here " that works with both cgroupv1 and cgroupv2 unlike the deprecated limits.network.priority instance setting which only worked with cgroupv1".